### ISSUE TITLE
Add lesson on contributing to the software

### ISSUE THREAD

**Issue by ibab** [lhcb/second-analysis-steps#6]
The second analysis steps seem to be a good place for a lesson on contributing to the LHCb software.

Can start with some basic contribution guidelines ("Everyone is encouraged to contribute. You will receive help, etc.")

Should answer questions
- What can I contribute?
- Where can I contribute?
- What are the rules?
- What is the _process_?

For the contribution process, we could describe a merge request based workflow in Gitlab.

**Comment by saschastahl** [comment]
I think as long as we have not switched to Gitlab (which should be done rather sooner than later) it should also show how to contribute to our current svn repository. Otherwise people are still in the dark and won't start contributing.

**Comment by ibab** [comment]
Yes, we might very well have only SVN for now and change the lesson to use Gitlab later.

**Comment by kdungs** [comment]
I would want to have both from the start. Especially judging from the discussions at the computing workshop, change is necessary **now**.

**Comment by saschastahl** [comment]
Yes, but we also need to get people involved **now** :).

**Comment by kdungs** [comment]
Fully agree which is why I would go with teaching both. In an ideal world, we'd only teach Git+Git{hub,lab} but I acknowledge that this is not possible atm.
